File: vocabulary.learning.app/src/main/java/com/vocabulary/learning/app/model/AllMeanings.java
package com.vocabulary.learning.app.model;

import com.vocabulary.learning.app.enums.MeaningTextType;
import com.vocabulary.learning.app.enums.MeaningType;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDateTime;


@Data
@NoArgsConstructor
@Setter
@Getter
@ToString
public class AllMeanings {
    private Integer id;
    private String title;
    private MeaningType meaningType;
    private String text;
    private MeaningTextType meaningTextType;
    private String script;
    private String meanings;
    private LocalDateTime createdTimeStamp;
    private LocalDateTime updatedTimeStamp;
}

File: vocabular-learning-app-ui/src/app/models/all-meanings.ts
import { MeaningTextType } from "./meaning-text-type";
import { MeaningType } from "./meaning-type";

export class AllMeanings {

    public id: number = 0;
    public title: string = "";
    public meaningType: MeaningType = MeaningType.ARTICLE;
    public text: string = "";
    public meaningTextType: MeaningTextType = MeaningTextType.TEXT;
    public script: string = "";
    public meanings: string = "";
    public createdTimeStamp: Date | undefined;
    public updatedTimeStamp: Date | undefined;
}
